test/Microsoft.TestPlatform.PerformanceTests/TranslationLayer/DiscoveryPerfTests.cs

```diff
@@ -61,5 +61,47 @@ public void DiscoverNunit10K()
 
             this.PostTelemetry("DiscoverNunit10K", this.discoveryEventHandler2.Metrics);
         }
+
+        [TestMethod]
+        [TestCategory("TelemetryPerf")]
+        public void DiscoverMsTest10KWhenDefaultAdaptersSkipped()
+        {
+            var testAssemblies = new List<string>
+                                     {
+                                         GetPerfAssetFullPath("MSTestAdapterPerfTestProject", "MSTestAdapterPerfTestProject.dll"),
+                                     };
+
+            this.vstestConsoleWrapper.DiscoverTests(testAssemblies, this.GetDefaultRunSettings(), new TestPlatformOptions() { CollectMetrics = true, SkipDefaultAdapters = true }, this.discoveryEventHandler2);
+
+            this.PostTelemetry("DiscoveryMsTest10K", this.discoveryEventHandler2.Metrics);
```

Comment:
Change `DiscoveryMsTest10K` to `DiscoverMsTest10KWithDefaultAdaptersSkipped`.

Reply:
done